megamek/src/megamek/client/ui/swing/tileset/TilesetManager.java

```diff
@@ -583,7 +583,9 @@ public synchronized void loadImage(Entity entity, int secondaryPos) {
 
         IPlayer player = entity.getOwner();
 
-        Camouflage camouflage = entity.getCamouflageOrElse(player.getCamouflage());
+        Camouflage camouflage = player != null ?
+                entity.getCamouflageOrElse(player.getCamouflage()) :
+                new Camouflage();
```

Comment:
I'd prefer the following, which adds brackets and removes the need to mentally swap the trinary, especially as part of camouflage.
```suggestion
        Camouflage camouflage = (player == null) ? new Camouflage()
                : entity.getCamouflageOrElse(player.getCamouflage());
```